testng-eclipse-plugin/src/main/org/testng/eclipse/launch/TestNGLaunchConfigurationDelegate.java

```diff
@@ -355,18 +355,6 @@ protected VMRunnerConfiguration createVMRunner(
   }
 
   @Override
-  public String getVMArguments(ILaunchConfiguration configuration)
-      throws CoreException {
-    StringBuilder vmArgs = new StringBuilder();
-    vmArgs.append(super.getVMArguments(configuration));

```

Comment:
the 1st call retrieves `org.eclipse.jdt.launching.VM_ARGUMENTS` and append to testng JVM process